Source/ImPlotWidgets/Public/LineFilledGraphWidget.h

```diff
@@ -22,5 +31,7 @@ class IMPLOTWIDGETS_API SFilledGraphWidget : public SImPlotWidget
 
 	mutable TArray<bool> ShowLine;
 	TArray<FImPlotData> DataSets;
+	TArray<float> YMaxValues;
```

Comment:
You won't need this `YMaxValues` if you follow (this comment) as it's only used temporarily to work out what the best fit for the graphs is